packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { PropTypes } from 'react';
+import moment from 'moment';
+import TextMask from 'react-text-mask';
+import TimePipe from './TimePipe';
+import './TimeInput.scss';
+
+const propTypes = {
+  /**
+   * A moment object to use as the default initial time.
+   */
+  defaultTime: PropTypes.object,
+    /**
+   * A callback function to execute when a time value is entered.
+   */
+  onChange: PropTypes.func,
+};
+
+const defaultProps = {
+  defaultTime: null,
```

Comment:
Yup, these props have been updated to default to `undefined`
https://github.com/cerner/terra-core/blob/terra_time_input/packages/terra-time-input/src/TimeInput.jsx#L30-L32